### ISSUE TITLE
Make attribution links work

### ISSUE THREAD

**Issue by Razzeee** [bromix/plugin.video.youtube#248]
These links from youtube emails can't be send directly to yatze.
You first need to open the video, which is really a pain.

```
08:46:47 T:1442837568  NOTICE: Creating Demuxer
08:46:48 T:1442837568   ERROR: Open - error probing input format, http://www.youtube.com/attribution_link?a=ceH_XxDC7jU&u=/watch_videos%3Fvideo_ids%3DxJHzzCd_Hgs%252CvfBLzqKwVKM%252Cvc-iMvzVbL0%26no_autoplay%3D1%26feature%3Dem-subs_digest%26title%3DNeueste%2BVideos%2Bvon%2BKotaku%2Bvom%2B25.04.2015
08:46:48 T:1442837568   ERROR: OpenDemuxStream - Error creating demuxer
```

**Comment by bromix** [comment]
Please inform the developer of Yatse...Yatse has to transform these urls. The addon isn't even called by KODI because of the **http** scheme in the URI.

**Comment by Razzeee** [comment]
Thank you for clearing that up, wasn't sure about it!

**Comment by bromix** [comment]
NP ;) The developer of Yatse will/should know what to do...the ID of the video is in there.

**Comment by Razzeee** [comment]
Yeah, it should be pretty straight forward. But i probably wouldn't want to regex it, following the redirect might be better. (thats up to the yatze devs ;) )

**Comment by Tolriq** [comment]
@bromix after some searches this is not so simple.

Those links contains multiple videos ids that are tied to a virtual playlist but I do not have the playlist id in those links.

```
http://www.youtube.com/attribution_link?a=ceH_XxDC7jU&u=/watch_videos%3Fvideo_ids%3DxJHzzCd_Hgs%252CvfBLzqKwVKM%252Cvc-iMvzVbL0%26no_autoplay%3D1%26feature%3Dem-subs_digest%26title%3DNeueste%2BVideos%2Bvon%2BKotaku%2Bvom%2B25.04.2015
```
Will lead to :
```
https://www.youtube.com/watch?v=xJHzzCd_Hgs&list=TLs6gmLMoSkcg
```
I can for sure only handle the first video, but it's not really Yatse job to queue all the other videos.
It's also not Yatse job to try to open the page, follow the redirection then get list id.

Maybe there's need for a new endpoint to handle those ?

**Comment by bromix** [comment]
Mhmm....I see...somehow I think about that => #176 :) Just push the whole thing to KODI and let the magic happen there. I thought there was only one video ID in it.

**Comment by Tolriq** [comment]
Well #176 would still need some endpoint in this particular plugin, not sure it would be the job of the resolver plugin to do very complex work to get the necessary data to push to final plugin ?

**Comment by bromix** [comment]
True...I have to take a look at these **attribute links**, I thought they only have one video ID in it :)
At the end (maybe) a new endpoint but some addon to relay such urls would be very helpful for both of us :)

**Comment by Tolriq** [comment]
Well so seems like you are interested in my idea :)

Just tell me if you are OK and need anything :)

**Comment by bromix** [comment]
@Tolriq 
you may take a look at this....http://forum.kodi.tv/showthread.php?tid=228877

I tried something...everything should be explained in the thread.

**Comment by Razzeee** [comment]
There still seem to be some urls that are not working see this
```
15:16:26 T:1612321856   ERROR: Open - error probing input format, http://www.youtube.com/attribution_link?a=e7cpclhuOa4&u=/watch%3Fv%3DyBbpWauMHXo%26feature%3Dem-subs_digest
```

**Comment by bromix** [comment]
We haven't solved this issue...first I need some other magic to resolve this clean.
see: http://forum.kodi.tv/showthread.php?tid=228877

**Comment by Razzeee** [comment]
Well tolriq has something in place for this case probably a dirty workaround for now
See http://yatse.tv/redmine/issues/33
Should have posted there, sorry

**Comment by bromix** [comment]
It should be possible for the addon to resolve these URLs but I would love to see it working with a common solution. Otherwise I have to create new dirty endpoints just for these kind of URLs.

**Comment by Razzeee** [comment]
Yes and I'm fully there with you.
But if it's not working for some and some not the I think it should get fixed for now. Even if it is not with the prefered way.
